ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -366,7 +366,7 @@ EOF
 
   cp "${dir}/bootstrap.ign" "${config_dir}/bootstrap_initial.ign"
   # We're using ancient fcct as glibc in installer container is 1.28
-  curl -sL https://github.com/coreos/butane/releases/download/v0.7.0/fcct-x86_64-unknown-linux-gnu >/tmp/fcct && chmod ug+x /tmp/fcct
+  curl -sL https://github.com/coreos/butane/releases/download/v0.7.0/fcct-"$(uname -m)"-unknown-linux-gnu >/tmp/fcct && chmod ug+x /tmp/fcct
```

Comment:
Done.

Reply:
oh, I was wrong - we do need these quotes (but not below):
```
 In .//ci-operator/step-registry/ipi/install/install/stableinitial/ipi-install-install-stableinitial-commands.sh line 369:
  curl -sL https://github.com/coreos/butane/releases/download/v0.7.0/fcct-$(uname -m)-unknown-linux-gnu >/tmp/fcct && chmod ug+x /tmp/fcct
                                                                          ^---------^ SC2046: Quote this to prevent word splitting. 
```

Reply:
Done.